packages/@aws-cdk-testing/cli-integ/README.md

```diff
@@ -35,6 +35,17 @@ Each subdirectory contains one test **suite**, and in the development pipeline e
 
 Test suites are written as a collection of Jest tests, and they are run using Jest, using the code in the `lib/` directory as helpers.
 
+### Setup
+
+Building the @aws-cdk-testing package is not very different from building the rest of the CDK. However, If you are having issues with the tests, you can ensure your enviornment is built properly by following the steps below:
+
+```shell
+yarn install # Install dependencies
+npx lerna run build --scope=aws-cdk # Build the CDK cli
+yarn build # Build the @aws-cdk-testing/cli-integ package
+../../../scripts/align-version.sh # Align the versions of CDK packages
```

Comment:
what is this `align-versions.sh` script lol. I have never run that before.

Reply:
it turns all the 0.0.0s into whatever the version of the cli is so the thing doesn't panic

Reply:
Oh shit. I wish I had known this a while ago. I've been manually working around this.

Reply:
I have never used this lol